ui/panels-plugin/src/plugins/time-series-chart/VisualOptionsEditor.tsx

```diff
@@ -105,6 +105,27 @@ export function VisualOptionsEditor({ value, onChange }: VisualOptionsEditorProp
           />
         }
       />
+      <OptionsEditorControl
+        label={'Palette'}
+        control={
+          <ToggleButtonGroup
+            color="primary"
```

Comment:
can you also fix the color in `ThresholdEditor` so the two toggle button groups look consistent?

Reply:
This will be deleted in follow up PR #1124 based on feedback from Mohit